pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -634,6 +642,33 @@ var _ = Describe("Pod Network", func() {
 
 			err = fmt.Errorf("failed to start DHCP server")
 			mockNetwork.EXPECT().StartDHCP(masq.vif, gomock.Any(), masq.bridgeInterfaceName, nil).Return(err)
+			// on masquerade we need to mock 'get'ting the bridge, so we can
+			// retrieve its mac address, which will be advertised
+			mockNetwork.EXPECT().CreateRADaemon(expectedSocketPath, api.DefaultBridgeName, api.DefaultVMIpv6CIDR, maxRetryNumber).Return(nil)
+			mockNetwork.EXPECT().LinkByName(api.DefaultBridgeName).Return(bridgeTest, nil)
+
+			err = masq.startDynamicIPServers(vmi)
+			Expect(err).To(HaveOccurred())
+		})
+
+		It("should fail when RA daemon failed", func() {
+			domain := NewDomainWithBridgeInterface()
+			vmi := newVMIMasqueradeInterface("testnamespace", "testVmName")
+			api.NewDefaulter(runtime.GOARCH).SetObjectDefaults_Domain(domain)
+			vmi.Spec.Domain.Devices.Interfaces[0].MacAddress = "de-ad-00-00-be-af"
+			driver, err := getPhase2Binding(vmi, &vmi.Spec.Domain.Devices.Interfaces[0], &vmi.Spec.Networks[0], domain, podInterface)
+			Expect(err).ToNot(HaveOccurred())
+			masq, ok := driver.(*MasqueradePodInterface)
+			Expect(ok).To(BeTrue())
+
+			masq.vif.Gateway = masqueradeGwAddr.IP.To4()
+			masq.vif.GatewayIpv6 = masqueradeIpv6GwAddr.IP.To16()
+
+			mockNetwork.EXPECT().StartDHCP(masq.vif, gomock.Any(), masq.bridgeInterfaceName, nil).Return(nil)
+
+			err = fmt.Errorf("failed to start RA daemon")
+			mockNetwork.EXPECT().CreateRADaemon(expectedSocketPath, api.DefaultBridgeName, api.DefaultVMIpv6CIDR, maxRetryNumber).Return(err)
+			mockNetwork.EXPECT().LinkByName(api.DefaultBridgeName).Return(nil, err)
```

Comment:
Mocking the `LinkByName` returning an error is enough, `CreateRADaemon` won't be even invoked.
Anyway, as the previous comments raised, it doesn't belong to this PR.